generator/sbpg/targets/resources/sbp_construct_template.py.j2

```diff
@@ -16,8 +16,11 @@
 
 from construct import *
```

Comment:
Yes, construct module is still used for message packing. I'll need to do a separate PR for that side and then we can remove the construct if so wanted.